packages/core/echo/indexing/src/index-schema.ts

```diff
@@ -4,7 +4,7 @@
 
 import { Event } from '@dxos/async';
 import { Resource } from '@dxos/context';
-import { type ObjectStructure } from '@dxos/echo-protocol';
+import { decodeReference, type ObjectStructure } from '@dxos/echo-protocol';
 import { EXPANDO_TYPENAME } from '@dxos/echo-schema';
```

Comment:
does this typename need to be updated as well?

Reply:
cc @richburdon 

Currently it is:

```ts
export const EXPANDO_TYPENAME = 'Expando';
```

Reply:
presumably it should be `dxos.org/type/Expando` and we might as well do that at the same time as the space properties